package.json

```
@@ -1,18 +1,20 @@
 {
   "name": "shadow-study",
   "private": true,
-  "version": "1.0.0",
+  "version": "1.1.0",
   "type": "module",
   "scripts": {
-    "start": "DEV_SERVER=1 vite",
+    "start": "cross-env DEV_SERVER=1 vite",
```

I see `DEV_SERVER=1` is unused in this repo and something from other internal repos. You can also remove the `cross-env` dependency.

```suggestion
    "start": "vite",
```

---

Removed unused DEV_SERVER and cross-env.